src/ruby/ext/grpc/rb_channel.c

```diff
@@ -175,11 +175,13 @@ static VALUE grpc_rb_channel_init(int argc, VALUE *argv, VALUE self) {
 
 /*
   call-seq:
-    insecure_channel = Channel:new("myhost:8080", {'arg1': 'value1'})
-    creds = ...
-    secure_channel = Channel:new("myhost:443", {'arg1': 'value1'}, creds)
+    ch.connectivity_state       -> state
+    ch.connectivity_state(true) -> state
 
-  Creates channel instances. */
+  Indicates the current state of the channel, whose value is one of the
+  constants defined in GRPC::Core::ConnectivityStates.
+
+  It also tries to connect if the chennel is idle in the second form. */
```

Comment:
thanks, one more nit: I think line 192 needs to be changed to:

`rb_scan_args(argc, argv, "01", &try_to_connect);`

the change being +`&try_to_connect` and  -`try_to_connect`